models/reviews.js

```diff
@@ -1,3 +1,4 @@
+const { ident } = require("pg-format");
```

Comment:
Correct me if I'm wrong, but I don't think this is being used anywhere - so can be removed 👍

Reply:
Presumably auto added whilst typing something else (and autocorrecting). Had it happened a couple times, but this one seems to have slipped through the net. Thanks for the heads up on this!